### PR TITLE
Autotracking Bugfix: Make the model property on the controller tracked

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
~TODO:~
 - ~write failing test~
 - ~add tracked when auto-tracking is enabled~

when the tracked flag is enabled, the controller's model property will be tracked by default.

### CODE REVIEW

**Review by rwjblue — APPROVED** [review]
Seems good to me. 

@pzuraq - Any objections?

**Review by pzuraq — APPROVED** [review]

File: packages/ember/tests/routing/route_controller_integration_test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        this.add('route:home', HomeRoute);
+        this.add('controller:home', HomeController);
+        this.addTemplate('home', '<h3 class="derivedProperty">{{this.derivedProperty}}</h3>');
+
+        await this.visit('/home/2');
+        assert.equal(this.$('h3').text(), '2', 'the derived property matches the id');
+
+        await this.visit('/home/3');
+        assert.equal(this.$('h3').text(), '3', 'the derived property matches the id');
```
Comment (lifeart) [inline packages/ember/tests/routing/route_controller_integration_test.js]: do we need jquery selection?

Comment (NullVoxPopuli) [reply packages/ember/tests/routing/route_controller_integration_test.js]: probably not, but I saw other tests using it 🤷‍♂

Comment (lifeart) [reply packages/ember/tests/routing/route_controller_integration_test.js]: should it fail if jquery integration = false?

Comment (NullVoxPopuli) [reply packages/ember/tests/routing/route_controller_integration_test.js]: removed -- just using document.querySelector :-\

Comment (rwjblue) [reply packages/ember/tests/routing/route_controller_integration_test.js]: Ya, it’s an internal abstraction that we did during the “drip jQuery requirement” work

**Comment by NullVoxPopuli** [comment]
C.I. error:
```
unknown Statement of type "ClassDeclaration"
```
:(

**Comment by rwjblue** [comment]
I'm not sure exactly why this is an issue, but regenerator-runtime is throwing that error. Specifically, it doesn't like the inline class declarations within the test function. 

There are two "easy" choices:

* Rewrite to avoid using async / await -- This will almost certainly work, and should be _fairly_ simple.
* Rewrite `class Foo extends Controller {}` to `let Foo = class Foo extends Controller {}`-- I'm not 100% sure this will solve the issue, but it might.

**Comment by NullVoxPopuli** [comment]
> Rewrite to avoid using async / await -- This will almost certainly work, and should be fairly simple.

is it worth configuring the tests to support async/await?

**Comment by rwjblue** [comment]
They do support async / await in general, @pzuraq recently added that. The specific issue in this case is that (apparently) regenerator-runtime doesn't know what to do with a `ClassDeclaration`. The version in use, is older (0.11.1) so maybe it's fixed upstream already. I dunno.

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
@NullVoxPopuli 	- Can you squash?

**Comment by NullVoxPopuli** [comment]
squashed

**Comment by NullVoxPopuli** [comment]
idk what's up with the git history here...

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,10 @@
 - [#17940](https://github.com/emberjs/ember.js/pull/17940) [CLEANUP] Remove `sync` queue from @ember/runloop.
 - [#18026](https://github.com/emberjs/ember.js/pull/18026) Enabling featured discussed in 2019-05-03 core team meeting.
 
+### v3.10.1 (June 4, 2019)
```
Comment (NullVoxPopuli) [inline CHANGELOG.md]: where did this come from O.o

**Comment by NullVoxPopuli** [comment]
@rwjblue fixed history :tm: :)

**Comment by NullVoxPopuli** [comment]
looks like browserstack had a problem

**Comment by NullVoxPopuli** [comment]
can someone click re-run?

**Pull request merged**